packaging/installer/functions.sh

```diff
@@ -535,13 +518,8 @@ install_netdata_service() {
 			local ret=$?
 
 			if [ ${ret} -eq 0 ]; then
-				if [ -n "${service_cmd}" ]; then
-					NETDATA_START_CMD="service netdata start"
-					NETDATA_STOP_CMD="service netdata stop"
-				elif [ -n "${rcservice_cmd}" ]; then
-					NETDATA_START_CMD="rc-service netdata start"
-					NETDATA_STOP_CMD="rc-service netdata stop"
-				fi
+				portable_service netdata start
```

Comment:
This is different code. The current code just sets the command for use elsewhere, this one runs them. Looks like it will do something different.

Reply:
The old code, sets NETDATA_START_CMD and NETDATA_STOP_CMD while `portable_service netdata start` and `portable_service netdata stop` execute a stop start.

What do i miss?

Reply:
oops sorry, this is resolved at a later commit. I will resolve the conversation, i thought its still open